propagators/datadog/hs-opentelemetry-propagator-datadog.cabal

```diff
@@ -0,0 +1,91 @@
+cabal-version: 2.4
+
+name: hs-opentelemetry-propagator-datadog
+version: 0.0.0.0
+author: Kazuki Okamoto (岡本和樹)
+maintainer: kazuki.okamoto@herp.co.jp
+
+common common
+  build-depends: base >= 4 && < 5
+  ghc-options: -Wall
+  if impl(ghc >= 8.0)
+    ghc-options: -Wcompat
+  default-language: Haskell2010
+
+library
+  import: common
+  hs-source-dirs: src
+  exposed-modules: OpenTelemetry.Propagator.Datadog
+                   OpenTelemetry.Propagator.Datadog.Internal
+  build-depends: bytestring,
+                 hs-opentelemetry-api,
+                 hs-opentelemetry-sdk,
+                 http-types,
+                 primitive,
+                 text
+  ghc-options: -Wcompat
+               -Wno-name-shadowing
+  if impl(ghc >= 6.4)
+    ghc-options: -Wincomplete-record-updates
```

Comment:
Is this automatically generated from some tool?

Reply:
This is from my project template https://github.com/kakkun61/haskell-project-template.